swift/ql/lib/codeql/swift/generated/Callable.qll

```diff
@@ -11,6 +12,9 @@ import codeql.swift.elements.Element
 import codeql.swift.elements.decl.ParamDecl
 
 module Generated {
+  /**
+   * INTERNAL: Do not use. Use the wrapper class under `elements` instead.
+   */
```

Comment:
This is what I'm talking about.  I think if we're putting a warning here (and I'm not entirely convinced we need one) it needs to ask you to use `Callable` rather than `Generated::Callable`, without implying that you should not use the contents of this class.

Reply:
Do you think this wording could be improved? Maybe
```codeql
  /**
   * INTERNAL: Do not use. Rather than `Generated::Callable`, use the subclass `Callable`
   * from `codeql.swift.elements.Callable`, where the following predicates are available.
   */
```

Reply:
hmm, but that pushes to import `codeql.swift.elements.Callable`, which should not be necessary. Maybe `codeql.swift.elements` should be mentioned.

Reply:
My attempt (I'm specifically avoiding the phrase "Do not use" here because I feel that is too broad):
```
/**
 * INTERNAL: Do not reference the `Generated::Callable` class directly.  Use the subclass
 * `Callable`, where the following predicates are available.
 */
```

Reply:
ok, nice one, I'll update the PR after lunch

Reply:
Done. After all, I did not specify _which_ lunch :see_no_evil: